Fix docs for CodePush and Appcenter missing params (#4676)
getsentry/sentry-docs: src/platforms/react-native/manual-setup/codepush.mdx

sentry-cli react-native appcenter {APP} ios ./build/CodePush --deployment {DEPLOYMENT} --dist {DIST}
```

For non-macOS devices building for iOS, you will need to pass extra parameters that can't be computed automatically (`--bundle-id` and `--version-name`).

```bash
sentry-cli react-native appcenter {APP} ios ./build/CodePush --deployment {DEPLOYMENT} --dist {DIST} --bundle-id {BUNDLE_ID} --version-name {VERSION_NAME}
```

> In case you don't want to use the automatically computed release format, you will need to pass the `--release-name` parameter.

Reviewer: ```suggestion
If you don't want to use the automatically computed release format, you will need to pass the `--release-name` parameter.
```
